src/models/multiBarChart.js

```diff
@@ -334,12 +334,14 @@ nv.models.multiBarChart = function() {
 
     multibar.dispatch.on('elementMouseover.tooltip', function(evt) {
         evt.value = chart.x()(evt.data);
-        evt['series'] = {
-            key: evt.data.key,
-            value: chart.y()(evt.data),
-            color: evt.color
-        };
-        tooltip.data(evt).hidden(false);
+        if (!chart.stacked() || (evt.data.size > 0 || evt.data.size == undefined)) {
```

Comment:
I can actually get it in the `multiBarChart.html` and the `multiBarChart2.html` examples. I guess the problem is only with some browsers. The `series` field actually disappears when I use Safari, but it's there when I'm using Chrome or Firefox. Anyway I always get the `evt.data.y` and we can use that one for the same purpose. I'll do another commit.